City-Bureau/city-scrapers-bismarck
# 🏗️ Build mixin + spiders: bcc

## What's this PR do?
Adds a mixin and spiders to scrape multiple meetings from the website of Bismarck City Commission The mixin that handles the core scraping logic is called `bcc`.

[Note: This PR was prematurely merged and was promptly reverted. Please refer to #7 for the final, reviewed version of these changes)

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spiders:
```
scrapy list | grep 'bcc' | xargs -I {} scrapy crawl {} -O {}.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect the CSVs to ensure the data looks valid.

## Are there any smells or added technical debt to note?
